lib/server-builder.js

```diff
@@ -74,7 +74,7 @@ class ServerBuilder {
 
   getCommand () {
     const cmd = system.isWindows() ? 'gradlew.bat' : './gradlew';
-    const buildProperty = (key, value) => value ? `-P${key}=${value}` : null;
+    const buildProperty = (key, value) => value ? `-P${key}="${value}"` : null;
```

Comment:
I would rather check explicitly if the corresponding value already contains quotes and throw an error.

Reply:
It seems like the root cause was `-PappiumTargetPackage="io.appium.android.apis"` etc was wrapped by `'` in the command. So, with `"`, the command will be like:

```
[Espresso] Unable to build Espresso server - Process ended with exitcode 1 (cmd: './gradlew '-PappiumTargetPackage="io.appium.android.apis"' app\:assembleAndroidTest')
```

But expected one should be `./gradlew -PappiumTargetPackage="io.appium.android.apis" app\:assembleAndroidTest`

Reply:
okay, i figured out.

The command will be `./gradlew '-PappiumKeyPassword=START&END'`.
Then, macOS and Linux have no issue. They handle "START&END" as a string.
But on Windows, `./gradlew '-PappiumKeyPassword=START&END'` will be `./gradlew -PappiumKeyPassword=START` and `END` in the command prompt. in the case, the argument should be `./gradlew -PappiumKeyPassword="START&END"`. In powershell, it should be `./gradlew -PappiumKeyPassword="START\&END"` (escape)

Reply:
@KazuCocoa The gradle [documentation](https://docs.gradle.org/current/userguide/command_line_interface.html) says 

> Options that accept values can be specified with or without = between the option and argument; however, use of = is recommended.

So we could maybe try to make the value to a separate arg instead of quoting the whole expression

Reply:
hm, I tried like the below command, but then, the separated arg was handled as a task.
```
% ./gradlew -PappiumKeyPassword "START&END" app:assembleAndroidTest 

FAILURE: Build failed with an exception.

* What went wrong:
Task 'START&END' not found in root project 'espresso-server'.

* Try:
Run gradlew tasks to get a list of available tasks. Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```


I've combined args as a command like:

```
    const gradlebuild = new SubProcess(cmd + ' ' + args.join(' '),[], {
      cwd: this.serverPath,
      stdio: ['ignore', 'pipe', 'pipe'],
    });
```
, then, the generated command was `./gradlew -PappiumTargetPackage="io.appium.android.apis" app:assembleAndroidTest ENOENT`. So the `ENOENT` prevent the command. Maybe i need to see `SubProcess`